DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -59,6 +62,10 @@ TopMonitor::TopMonitor( const edm::ParameterSet& iConfig ) :
   // Marina
   , nbjets_    ( iConfig.getParameter<unsigned int>("nbjets"))
   , workingpoint_(iConfig.getParameter<double>("workingpoint"))
+  , bjetPtCut_(iConfig.getParameter<double>("bjetPtCut"))
+  , bjetAbsEtaCut_(iConfig.getParameter<double>("bjetAbsEtaCut"))
```

Comment:
@sroychow please remove lines 65 and 66 (bjetPtCut_, bjetAbsEtaCut_). I removed them in my last commit, I think they sneaked back in after you resolved the conflict